src/docfx/build/Build.cs

```diff
@@ -13,7 +13,17 @@ internal static class Build
     {
         public static async Task Run(string docsetPath, CommandLineOptions options, ILog log)
         {
-            var config = Config.Load(docsetPath, options);
+            Config config;
+            try
+            {
+                config = Config.Load(docsetPath, options);
+            }
+            catch
+            {
+                // TODO: error handling
+                return;
+            }
```

Comment:
TryLoad?

Reply:
TryLoad is for loading config *might* exist, like resore. Rename to LoadIfExists to avoid confusion...

---
In reply to: [184842588](https://github.com/dotnet/docfx/pull/2702#discussion_r184842588) [](ancestors = 184842588)

Reply:
but i still dont want to add try catch here, what if we just throw the exception which is thrown by Config.Load?

---
In reply to: [185388015](https://github.com/dotnet/docfx/pull/2702#discussion_r185388015) [](ancestors = 185388015,184842588)

Reply:
Oops, forgot to remove this ... resolved

---
In reply to: [185437163](https://github.com/dotnet/docfx/pull/2702#discussion_r185437163) [](ancestors = 185437163,185435673,185388015,184842588)